capabilities/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/smartcontractkit/chainlink-evm/capabilities
+
+go 1.24.1
+
+require (
+	github.com/ethereum/go-ethereum v1.15.7
```

Comment:
Does it need to be this new? We are struggling to upgrade core already, so you'll be blocked if you bump this far.

Reply:
downgraded to match core